src/components/Services.jsx

```diff
@@ -0,0 +1,23 @@
+import React from 'react'
+import { serviceData } from '../servicedata'
+import './services.css'
+
+function Services () {
+  return (
+    <div className='flex flex-col w-full px-0 py-[94px] md:px-8 md:py-6'>
+        <h1 className='services mx-auto mb-6'>Services</h1>
```

Comment:
Comment(nb): This is okay for now, but make a card for yourself to use @NurkaAmre 's incoming `BaseHeading` component; that will keep the styling standardized.